datamap/mv/core/src/main/scala/org/apache/carbondata/mv/datamap/MVHelper.scala

```diff
@@ -152,6 +176,33 @@ object MVHelper {
           fields,
           fieldRelationMap,
           tableProperties)
+      if (timeSeriesFunction != null) {
+        if (null != mainTable && (parentTablesList.size() == 1)) {
```

Comment:
can you add a comment, why parentTablesList.size() == 1 check ? Need to throw exception if != 1

Reply:
This is in case of single parent table in query. Not supported for Join queries.